src/interfaces/messages/handlers/messages-get.ts

```diff
@@ -37,7 +36,7 @@ export class MessagesGetHandler implements MethodHandler {
     const messageCids = new Set(message.descriptor.messageCids);
 
     for (const messageCid of messageCids) {
-      const promise = this.messageStore.get(tenant, messageCid)
+      const promise = this.storageController.MessageStore.get(tenant, messageCid)
```

Comment:
instead of exposing the `MessageStore` should we have a `getMessage`?
```
getMessage(tenant: string, messageCid: string) {
  return this.messageStore.get(tenant, messageCid);
}
```

Reply:
Addressed in https://github.com/TBD54566975/dwn-sdk-js/pull/338/commits/8a39de45aaa23185e406373a304c4ee9e4d68730